### ISSUE TITLE
feh modifies a passed filelist

### ISSUE THREAD

**Issue by kiteloopdesign** [derf/feh#698]
when passing a file with "-f" option in which the images are not listed as expected (one per line), feh decides to delete these files from the list

This is highly unexpected and dangerous behaviour, no tool should do this w/o confirmation or an extra switch (eg --edit)

**Comment by avlec** [comment]
This is the behaviour that is described in the man page.

**Comment by derf** [comment]
Indeed, though this issue makes me question whether this is sensible behaviour. I can understand that users would expect a filelist to remain unchanged unless they perform explicit actions to alter it.

I'll flag this as a bug for now and see how I can improve this.

**Comment by avlec** [comment]
One option that comes to mind that preserves existing behaviour is to chmod the file to remove write access. However, I am not sure that would be an acceptable solution

**Comment by kiteloopdesign** [comment]
derf, avlec, thanks for the answers

IMHO, it's not a sensible behaviour for any tool to alter a file in this way. Even more being the case of a filelist, which is just a means of providing the images to feh. This is, a user may expect an image browser/editor to make changes to the images, but not to a filelist.

Not sure why you provide this behaviour that avlec quoted on his reply, but, if you want to maintain it, I would suggest to break out the -f in two different options. That or I would require a new "--allowlistedit" flag

**Comment by umbra3275** [comment]
I'll second this as being unexpected behavior, despite being in the man page. I would have expected feh to just keep the internal list and leave the file as-is.
